### PR TITLE
FIX git url overflow styling

### PR DESCRIPTION
**Pull request by micmania1**

### CODE REVIEW

**Comment by micmania1** [comment]
git-repo-url-1

**Comment by halkyon** [comment]
What does it look like with the deploy key showing below?

**Comment by micmania1** [comment]
git-url-2

File: sass/style.sass [hunk 1]
```diff
@@ -1034,3 +1034,25 @@ td > .deployment-tag
...
+        line-height: 26px
+        padding: 5px
+        width: 18%
+        background: #ececec
+        border-right: 1px solid #dcdcdc
+        font-weight: bold
+        float: left
+
+    input[type="readonly"]
+        width: 82%
```
Comment (stojg) [inline sass/style.sass]: I got this feeling that the magic number 82% is to match the page title? Can that be documented / comment?

File: sass/style.sass [hunk 2]
```diff
@@ -1034,3 +1034,25 @@ td > .deployment-tag
...
+.create-page-git
+    border: 1px solid #dcdcdc;
+    display: block;
+    overflow: auto;
+
+    .title
+        display: block
+        line-height: 26px
+        padding: 5px
+        width: 18%
```
Comment (stojg) [inline sass/style.sass]: magic number `18%` can you document why?

**Pull request merged**